demisto_sdk/commands/secrets/tests/secrets_test.py

```diff
@@ -172,9 +180,17 @@ def test_remove_white_list_regex(self):
         shmoop
         155.165.45.232
         '''
-        file_contents = self.validator.remove_whitelisted_items_from_file(white_list, file_contents)
+        file_contents = self.validator.remove_whitelisted_items_from_file(file_contents, white_list)
         assert white_list not in file_contents
 
+    def test_remove_whitelisted_items_from_file_escaped_whitelist(self):
+        white_list = '***.url'
+        file_contents = '''
+        Random and unmeaningful file content
+        '''
+        file_contents = self.validator.remove_whitelisted_items_from_file(file_contents, white_list)
+        assert file_contents
```

Comment:
What are we checking here?

Reply:
The secret contains `***.url` which is an invalid regex that causes the system to raise an exception.